docs/en/transform/replace.md

```diff
@@ -1,16 +1,17 @@
-# Json
+# Replace
 
 ## Description
 
-Examines string value in a given field and replaces substring of the string value that matches the given string literal or regexes with the given replacement.
+Examines string value in a given field and replaces substring of the string value that matches the given string literal
+or regexes with the given replacement.
 
 :::tip
 
-This transform **ONLY** supported by Spark.
+This transform can supported by Spark and Flink,There are slight differences between the two engine configuration items.
 
 :::
 
-## Options
+## Options(Spark)
```

Comment:
It's better to use `<Tabs>` to manage the parameters in different engine. And it would be better to implement this plugin with same parameters in different engine.